Tests/Marketplace/Tests/zip_packs_test.py

```diff
@@ -102,10 +102,11 @@ def test_copy_zipped_packs_to_artifacts(self, mocker):
         """
         import shutil
         zipped_packs = [{'Slack': 'content/packs/Slack/1.0.1/Slack.zip'}]
+        artifatcs_path = 'dummy_path'
         mocker.patch.object(shutil, 'copy', side_effect=None)
         mocker.patch('os.path.exists', return_value=True)
 
-        copy_zipped_packs_to_artifacts(zipped_packs)
+        copy_zipped_packs_to_artifacts(zipped_packs, artifatcs_path)
```

Comment:
You have a typo in the variable name - it should be `artifacts_path`.

Reply:
thanks, fixed it.